cms/djangoapps/contentstore/views/component.py

```diff
@@ -37,7 +37,7 @@
 
 from ..utils import get_lms_link_for_item, get_sibling_urls, reverse_course_url
 from ..helpers import get_parent_xblock, is_unit, xblock_type_display_name
-from cms.djangoapps.contentstore.xblock_services.xblock_service import (
+from cms.djangoapps.contentstore.xblock_storage_handlers.view_handlers import (
     add_container_page_publishing_info,
     create_xblock_info,
     load_services_for_studio,
```

Comment:
In cms/contentstore/views/block.py, you are importing functions directly from xblock_storage_handlers:
```python
from cms.djangoapps.contentstore.xblock_storage_handlers import (
    handle_xblock,
    create_xblock_info,
    ...
)
```
But here, you are importing through the sub-packages of xblock_storage_handlers:
```python
from cms.djangoapps.contentstore.xblock_storage_handlers.view_handlers import (
    add_container_page_publishing_info,
    create_xblock_info,
    ...
)
```
I think either is OK, but I suggest choosing one and sticking with it.